app/models/profile.rb

```diff
@@ -24,10 +32,19 @@ class Profile < ActiveRecord::Base
   end
   dragonfly_accessor :background_image
 
+  has_attached_file :tombo_avatar
+  has_attached_file :tombo_background_image
+
   belongs_to :user
 
   validates :description, length: { maximum: 150 }
   validates :name, presence: true
+  validates :tombo_avatar, attachment_content_type: {
+                             content_type: /\Aimage/
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.